test/integration/testPlotCustodianManager.js

```diff
@@ -1343,6 +1349,7 @@ contract('PlotCustodianManager', (accounts) => {
           await this.plotCustodianManager.claimGaltSpaceReward(this.aId, { from: galtSpaceOrg });
 
           let res = await this.plotCustodianManagerWeb3.methods.getApplicationById(this.aId).call();
+          _.extend(res, await this.plotCustodianManagerWeb3.methods.getApplicationFinanceById(this.aId).call());
```

Comment:
what does this mean?

Reply:
its need because of some application fields was moved to getApplicationFinanceById function, so lodash extend function added this fields to base application object

Reply:
it looks too confusing, any reason why we use this?

Reply:
because tests will fail without this. Method getApplicationById gives not enough fields for tests passing.